main.go

```diff
@@ -132,93 +133,61 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getActiveMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
-	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
-	URL := strings.Join(strURL, "")
-	milestones := map[string]string{}
-
-	m, err := gitlabGet(URL, token)
-	if err != nil {
-		return milestones, err
-	}
-
-	for _, milestone := range m {
-		milestones[milestone.Title] = milestone.DueDate
-	}
-
-	return milestones, nil
+func getActiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "state=active"
+	return getMilestones(baseURL, token, projectID, state)
 }
 
 // Get and return inactive milestones
-func getInactiveMilestones(baseURL string, token string, projectID string) (map[string]int, error) {
-	strURL := []string{baseURL, projectID, "/milestones?state=closed&per_page=100"}
-	URL := strings.Join(strURL, "")
-	milestones := map[string]int{}
-
-	m, err := gitlabGet(URL, token)
-	if err != nil {
-		return milestones, err
-	}
-
-	for _, milestone := range m {
-		milestones[milestone.Title] = milestone.ID
-	}
-
-	return milestones, nil
+func getInactiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "state=inactive"
```

Comment:
Switch from `state=closed` to `state=inactive` voluntary ?

Reply:
Based on the Milestones API, it looks like we should probably use ```state=closed```? https://docs.gitlab.com/ce/api/milestones.html#list-project-milestones

Reply:
Yep, that's how it was before your commit :^)

Reply:
Thanks - I’ll change it back :)